pyanaconda/installation.py

```diff
@@ -319,6 +320,16 @@ def run_pre_install():
         late_storage.append_dbus_tasks(STORAGE, [conf_task])
         installation_queue.append(late_storage)
 
+    # Add installation tasks for the Subscription DBus module
+    # - to see if this is appropriate for the current environment we check if
+    #   the Subscription module is running via a helper function
+    if check_subscription_module_available():
+        subscription_config = TaskQueue("Subscription configuration",
```

Comment:
I don't think this queue is connected to the installation queue.

Reply:
Fixed during the move.